cfgov/unprocessed/js/modules/FlyoutMenu.js

```diff
@@ -53,11 +53,12 @@ function FlyoutMenu( element ) { // eslint-disable-line max-statements, no-inlin
   var _isExpanded = false;
   var _isAnimating = false;
 
-  var _expandTransition;
-  var _collapseTransition;
-  var _expandTransitionMethod;
+  var _expandTransition = null;
+  var _expandTransitionMethod = null;
   var _expandTransitionMethodArgs = [];
-  var _collapseTransitionMethod;
+
+  var _collapseTransition = null;
```

Comment:
Ah! I missed that we had undefined in standard-type. Sure, that works. I just didn't want three possible states. So what would be an appropriate use of null in authoring? From the ES spec I'm not seeing how it doesn't fit as a placeholder in this scenario.